src/components/Dashboard/DashboardBannerRow.tsx

```diff
@@ -1,17 +1,25 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect, useState, useMemo } from "react";
+import { Link } from "react-router-dom";
 import { NextToJump } from "../../types/meets";
 import utils from "../../utils";
+import { Config } from "../../types/config";
+import { getVenuefromConfig } from "../../utils/config";
 
 const INTERVAL = 1000;
 
 type Props = {
   meet: NextToJump;
+  config?: Config;
 };
 
-export const DashboardBannerRow: React.FC<Props> = ({ meet }) => {
+export const DashboardBannerRow: React.FC<Props> = ({ meet, config }) => {
   const [timeString, setTimeString] = useState(
     utils.formatting.formatTimeToHMS(meet.jumperRaceStartTime)
   );
+  const venueLocation = useMemo(
+    () => getVenuefromConfig(meet.meeting.jumperMeetingName, config),
```

Comment:
`getVenueFromConfig`

Reply:
also this probably doesn't need to be memoised as it's a linear search through an object